liftoff/1.6.3/Dockerfile

```diff
@@ -0,0 +1,72 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the folowing stage. We use `app` for the production image
+FROM ubuntu:bionic as app
+
+#https://bitbucket.org/genomicepidemiology/plasmidfinder/issues/10/version-number
+# ARG sets environment variables during the build stage
+ARG LIFTOFF_VER="1.6.3"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+# Optional, but highly recommended
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="liftoff"
+LABEL software.version=$LIFTOFF_VER
+LABEL description="A tool that accurately maps annotations in GFF or GTF between assemblies of the same, or closely-related species."
+LABEL website="https://github.com/agshumate/Liftoff"
+LABEL license="https://github.com/agshumate/Liftoff/blob/master/LICENSE.md"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# RUN executes code during the build
+# Install dependencies via apt-get or yum if using a centos or fedora base
+RUN apt-get update -y && apt-get install -y \
+    build-essential \
+    zlib1g-dev \
+    libncurses5-dev \
+    libgdbm-dev \
+    libnss3-dev \
+    libssl-dev \
+    libreadline-dev \
+    libffi-dev \
+    libsqlite3-dev \
+    libbz2-dev \
+    liblzma-dev \
+    wget && \
+    rm -rf /var/lib/apt/lists/* && apt-get autoclean
+
+#Install Python from source
+RUN wget https://www.python.org/ftp/python/3.8.0/Python-3.8.0.tgz && \
```

Comment:
Is there a reason why the python3 and pip from apt-get won't work? According to their bioconda package, anything 3.6+ should be fine.

Reply:
Yes there were some dependency issues with some of the python libraries. Numpy and Pysam would not install with the latest versions of pip3 and python that you get with apt-get. You can upgrade pip using pip but then it dosen't like the version of python. I guess there is a release of pip that would work with the apt-get version of python but that seemed like a headache trying to guess which one would work.